src/utilities/fe_dofs.jl

```diff
@@ -0,0 +1,116 @@
+type FEField
```

Comment:
Maybe a FEField should have a function space. In that way you could make for example a pressure field with linear base functions and a displacement field with quadratic basefunctions.